firmware/arduino_libraries/DAC_MCP49xx/DAC_MCP49xx.h

```diff
@@ -0,0 +1,61 @@
+#ifndef _DAC_MCP49xx_H
+#define _DAC_MCP49xx_H
+
+#include <SPI.h>
+#include <Arduino.h>
+#include <inttypes.h>
+
+#ifndef _SPI_H_INCLUDED
+#error Please include SPI.h before DAC_MCP49xx.h!
+#endif
+
+// Microchip MCP4901/MCP4911/MCP4921/MCP4902/MCP4912/MCP4922 DAC driver
```

Comment:
Can we explicitly call out the license here please? 

Or is it ok that .cpp source file has the full license explanation? @nathanaschbacher.